CONTRIBUTING.md

```diff
@@ -113,11 +113,8 @@ Other Tips
 - To skip only the tests running on Travis CI use ``[skip travis]``.
   This will still execute CircleCI.
 
-- If your commit makes substantial changes to the documentation but no code
-  changes, then you can use ``[ci skip]``, which will skip all CI except RTD,
-  where the documentation is built. The exception to this rule
-  is when your changes to documentation include code snippets that need to
-  be tested using ``doctest``.
+- If your commit makes substantial changes to the documentation but none of those changes include code snippets, then you can use ``[ci skip]``, which will skip all CI except RTD,
```

Comment:
Can you please break this long line into multiple shorter lines to be more consistent with the rest of the document? Thanks!

Reply:
Done!